cmd/cluster-agent-cloudfoundry/app/app.go

```diff
@@ -201,6 +202,14 @@ func run(cmd *cobra.Command, args []string) error {
 		return log.Errorf("Error while starting agent API, exiting: %v", err)
 	}
 
+	if config.Datadog.GetBool("cluster_agent.admission_controller.enabled") {
+		log.Info("Starting admission controller server")
+
+		if err = admissioncontroller.StartServer(); err != nil {
+			return log.Errorf("Error while starting admission controller server, exiting: %v", err)
+		}
+	}
+
 	// Block here until we receive the interrupt signal
```

Comment:
I think this code doesn't warrant an explanation, this is a very common pattern in Go.

Reply:
I'm leaning towards leaving this untouched as it is existing code not introduced in this PR, and the change wouldn't have a large impact.